### ISSUE TITLE
Comments: Need Error Message on Length Exceeding a 1000 Characters

### ISSUE THREAD

**Issue by blu3knight** [nextcloud/deck#1059]
**Describe the bug**
No Error Message is displayed in the comments it just keeps spinning, had to go in to logs to figure out that this was because Length is > 1000 Characters

**To Reproduce**
Have more then a 1000 Characters message

**Expected behavior**
Error message of some kind to appear

**Screenshots**

**Client details:**
 - OS: MAC OSX
 - Browser Safari
 - Version Version 12.1
 - Device: Laptop

<details>
<summary>Server details</summary>

**Operating system**:
Linux / Gentoo

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
7.1

**Nextcloud version:** (see Nextcloud admin page)

**Where did you install Nextcloud from:**

**Signing status:**

```
Login as admin user into your Nextcloud and access
http://example.com/index.php/settings/integrity/failed
paste the results here.
```

**List of activated apps:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ app:list
from within your Nextcloud installation folder
```

**Nextcloud configuration:**

```
If you have access to your command line run e.g.:
sudo -u www-data php occ config:list system
from within your Nextcloud installation folder

or

Insert your config.php content here
Make sure to remove all sensitive content such as passwords. (e.g. database password, passwordsalt, secret, smtp password, …)
```

**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...

</details>

<details>
<summary>Logs</summary>

#### Nextcloud log (data/nextcloud.log)
```
Insert your Nextcloud log here
```

#### Browser log
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```

</details>

**Comment by juliushaertl** [comment]
Fixed in #1508

**Comment by smehrbrodt** [comment]
I just tried to add a comment which has about 480 chars, but contains a number of links.
I still got the error message, maybe because the links were expanded to html?

Is there a technical reason for this limitation?
